lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -127,6 +127,17 @@ def set_z(self, z):
         self.stale = True
 
     def set_3d_properties(self, z=0, zdir='z'):
+        """
+        Set the *z* position and direction of the text.
+
+        Parameters
+        ----------
+        z : float
+            The z-position in 3D space.
+        zdir : {'x', 'y', 'z', 3-tuple}
+            The direction of the text. Default: ``'z'``.
```

Comment:
```suggestion
        zdir : {'x', 'y', 'z', 3-tuple}, default: 'z'
            The direction of the text.
```
Also a number of following places.